niljs/src/types/ITransaction.ts

```diff
@@ -1,10 +1,14 @@
-import type { ValueOf } from "@chainsafe/ssz";
-import type { SszTransactionSchema } from "../encoding/ssz.js";
-
 /**
  * The interface for the transaction object. This object is used to represent a transaction in the client code.
  * It may differ from the actual transaction object used inside the network.
  */
-interface ITransaction extends ValueOf<typeof SszTransactionSchema> {}
-
-export type { ITransaction };
+export type ITransaction = {
+  feeCredit: bigint;
+  maxPriorityFeePerGas: bigint;
+  maxFeePerGas: bigint;
+  seqno: number;
+  chainId: number;
+  to: Uint8Array;
```

Comment:
Let's remove completely this module? It's not used anywhere

Reply:
Done.